### PR TITLE
fix(components): [button] style compatible issue with :not

### PR DESCRIPTION
**Pull request by dreambo8563**
downgrade :not(a,b) to :not(a):not(b) for compatiblity

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by dreambo8563** [comment]
@YunYouJun committed, you can find the changes now.

File: packages/theme-chalk/src/button.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@ $button-icon-span-gap: map.merge(
   vertical-align: middle;
   -webkit-appearance: none;
 
-  &:not(.is-text, .is-link, .#{$namespace}-button--text) {
+  &:not(.is-text):not(.is-link):not(.#{$namespace}-button--text) {
```
Comment (JeremyWuuuuu) [inline packages/theme-chalk/src/button.scss:43]: I think this originally means 

```css
&:not(.is-text), &:not(.is-link), &:not(.#{$namespace}-button--text) {}
```

**Comment by YunYouJun** [comment]
image

image

There seems to be a problem with the Pull bot.
This pr is again in a state where nothing has been modified.

**Comment by tolking** [comment]
> image
> 
> image
> 
> There seems to be a problem with the Pull bot. This pr is again in a state where nothing has been modified.

Is it possible that this is an empty PR ?

[button.scss](https://github.com/element-plus/element-plus/blob/dev/packages/theme-chalk/src/button.scss#L43) same as before

File: packages/theme-chalk/src/button.scss (hunk 1 above)
Comment (tolking) [reply packages/theme-chalk/src/button.scss]: `&:not(a, b)` same with `&:not(a):not(b)` , but the `&:not(a), &:not(b)` is completely different.

&:not(a), // just exclude a, but include b
&:not(b) // just exclude b, but include a

**Comment by YunYouJun** [comment]
> Is it possible that this is an empty PR ?

The user's pull bot automatically force push, so we should recreate a PR.

**Comment by dreambo8563** [comment]
> 

I have suspended the bot  and create new PR 
https://github.com/element-plus/element-plus/pull/7914 @YunYouJun

File: packages/theme-chalk/src/button.scss (hunk 1 above)
Comment (JeremyWuuuuu) [reply packages/theme-chalk/src/button.scss]: Right, my bad.

**Pull request merged**